### PR TITLE
First website sections

### PR DESCRIPTION
**Pull request by Jolak5**
Change the name of the feature branch. Branch contains the first two section of my portfolio website using this figma design - https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=hf3TvzQmi2NAAjbu-0

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav>
+      <a href=""><img src="images/Logo Placeholder.svg" alt="" /></a>
+
+      <a href=""><img src="images/Icon - Menu.svg" alt="" /></a>
+    </nav>
+    <header class="header-section">
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly refactor your code, and make sure the `nav` resides within the `header`. 😄 

_Please note that the `header` should only contain the top part of the webpage, like navigation, logo, search bar, etc. So you also have to refactor the main part of your webpage. Consider using a `section` tag instead._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          Hey there. I’m Joseph <br />
+          <span class="header-span"> I’m a software developer</span>
+        </h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social">
```
Comment (jasonfelice) [inline index.html:34]: - [ ] The webpage is not displaying the social icons. Kindly make sure everything is implemented correctly.

## Your Design:
image

## Figma Design:
image

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
```
Comment (jasonfelice) [inline styles.css:11]: - [ ] According to the project requirements, your webpage should have the background like the given Figma design:

image

File: README.md [hunk 4]
```diff
@@ -1 +1,187 @@
...
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (jasonfelice) [inline README.md:24]: - [ ] [OPTIONAL] The image is missing from your repository and hence compromising the aesthetics of your readme. You can retrieve this image from the source of this readme template. 😄

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @Jolak5, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No Linter errors✔️
Great project documentation✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Jolak5, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jolak5** [comment]
Merge feature branch into the master branch

**Pull request merged**